packages/devtools_app/lib/src/timeline/flutter/timeline_flame_chart.dart

```diff
@@ -90,10 +90,16 @@ class TimelineFlameChartState
   void _handleSelectedFrame() async {
     final TimelineFrame selectedFrame = _timelineController.selectedFrame.value;
     if (selectedFrame != null) {
+      // Bail early if the selection has not changed.
```

Comment:
nit: no real need to document this line. it is just the standard pattern of skipping setState when the state has not  changed.